app/views/provisioning_templates/_alerts.html.erb

```diff
@@ -1,3 +1,10 @@
 <% if @template.locked? -%>
   <%= locked_warning(::Template.find(params[:id])) %>
 <% end -%>
+
+<% if @template.errors.any? -%>
+<%= alert :header => _("Unable to save"),
+          :class  => 'alert-danger base in fade',
+          :text   => @template.errors.map { |e| ("<li>#{e.attribute}: #{e.message}</li>") }.join.html_safe
```

Comment:
Worth using some specific methods:
```
[13] pry(main)> hh.errors.messages
=> {:"interfaces.mac"=>["can't be blank"],
 :"interfaces.domain"=>["can't be blank"],
 :"interfaces.domain_id"=>["can't find domain with this id"],
 :operatingsystem_id=>["can't be blank"],
 :organization_id=>["can't be blank"],
 :location_id=>["can't be blank"],
 :architecture_id=>["can't be blank"]}
[14] pry(main)> hh.errors.full_messages
=> ["Mac can't be blank",
 "Domain can't be blank",
 "Interfaces.domain can't find domain with this id",
 "Operatingsystem can't be blank",
 "Organization can't be blank",
 "Location can't be blank",
 "Architecture can't be blank"]
```

This example is for host object, but template object would behave the same.